classic/create.go

```diff
@@ -234,15 +234,34 @@ func customizeClassicChroot() error {
 		return fmt.Errorf("failed to cleanup classic /run dir: %s (%s)", err, output)
 	}
 
+	// Add SUDO_USER to sudo group in the chroot to ensure that
+	// sudo work for the calling user.
+	sudoUser := os.Getenv("SUDO_USER")
+	if sudoUser != "" {
+		// We need to use "runInClassicEnv" so that we get the
+		// bind mount of the /var/lib/extrausers directory.
+		// Without that the "SUDO_USER" will not exist in the chroot
+		if err := runInClassicEnv("usermod", "-a", "-G", "sudo", sudoUser); err != nil {
+			return fmt.Errorf("failed to add %s to the sudo users: %s", sudoUser, err)
+		}
+	}
+
```

Comment:
maybe I'm not understanding but shouldn't we copy the whole sudo group from outside to inside? it feels a bit strange to copy exactly the user that created the classic env